docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
 # 2. Quick Start Guide
 
 ## 2.1 Overview
-< Describe the general context around this walkthrough/lab >
+
+*Cobalt* is a tool for developers who are interested in reusing or contributing new cloud infrastructure as code patterns in template form. An actualized infrastructure as code pattern in Cobalt is called a *Cobalt Infrastructure Template* or *CIT* (/kɪt/). Cobalt Infrastructure Templates primarily rely on [*Terraform*](https://learn.hashicorp.com/terraform)'s HCL language in order to target a wide array of cloud providers.
+
+You can get pretty creative and build your own custom *CIT*s in order to use and/or contribute to Cobalt but we strongly recommend that you first complete this quick start guide. This guide is centered around our existing [*Azure Hello World CIT*](../infra/templates/az-hello-world/README.md "AZ Hello World - Cobalt Infrastructure Template") and should serve as your first Azure infrastructure deployment. In summary, completing this guide should be your first major step in familiarizing yourself with Cobalt and the *CIT* developer workflow. Happy templating! 😄
+
+> For a more general overview of Cobalt, please visit our main page: [READ ME](https://github.com/microsoft/cobalt/blob/master/README.md "Main Cobalt Read Me")
```

Comment:
If this is not in GitHub or the repo name changes the link will break. A relative reference will fix this
```suggestion
> For a more general overview of Cobalt, please visit our main page: [READ ME](../README.md "Main Cobalt Read Me")
```